### PR TITLE
Automate Download Native Videos

### PR DESCRIPTION
**Pull request by bilalawan321**
[LEARNER-8370](https://openedx.atlassian.net/browse/LEARNER-8370)

Verify Video tab showing following content:
        Videos as title
        course share icon
        Video TV title
        Video TV sub-title
        Video icon
        Download to device toggle
        Verify download to device permission dialogue contents:
        Allow button
        Deny Button
        Permission message 
        verify the following scenarios:
        check if download to device toggle off
        Click on toggle
        if allow/deny permission popup appears
        check to allow, deny, and permission elements
        Click on allow button
        check and click on the progress wheel
        check all videos are downloading
        wait for all videos to download
        check all videos are downloaded
        turn off toggle and check all video are deleted        
        check videos numbers with icons

### CODE REVIEW

**Comment by bilalawan321** [comment]
@jawad-khan can you please review this PR.?

File: tests/android/pages/android_video_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            True if video count is attached with every video
+            False if video count is not attached with any video
+        """
+
+        video_numbers_status = True
+        all_videos = self.global_contents.get_all_elements_by_id(
+            set_capabilities,
+            android_elements.video_dashboard_no_of_videos)
+
+        for video_elem in all_videos:
```
Comment (jawad-khan) [inline tests/android/pages/android_video_dashboard.py:68]: use all here

**Review by jawad-khan — APPROVED** [review]

File: tests/android/pages/android_video_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            True if all videos status are same
+            False if any single video status is not same
+        """
+
+        video_status = True
+        all_videos = self.global_contents.get_all_elements_by_id(
+            set_capabilities,
+            android_elements.video_dashboard_download_section)
+
+        for video_elem in all_videos:
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/android/pages/android_video_dashboard.py]: fixed

File: tests/android/pages/android_video_dashboard.py (hunk 1 above)
Comment (bilalawan321) [reply tests/android/pages/android_video_dashboard.py]: fixed

**Pull request merged**